jdaviz/configs/specviz/plugins/line_analysis/tests/test_line_analysis.py

```diff
@@ -162,7 +163,7 @@ def test_line_identify(specviz_helper, spectrum1d):
 
     # manually update redshift
     la_plugin.vue_line_assign()
-    assert la_plugin.selected_line_redshift == -1.0
+    assert_allclose(la_plugin.selected_line_redshift, 0.4594414354783614)
```

Comment:
Why does the value change for this test?

Reply:
I _think_ the old value was incorrect as the plugin was not updating the internal values (although we should confirm that independently outside of jdaviz)